live-examples/js-examples/intl/intl-relativetimeformat-prototype-formattoparts.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">var rtf1 = new Intl.RelativeTimeFormat('en', { numeric: 'auto' });
+console.log(rtf1.formatToParts(-1, 'day'));
+// expected output: [{ type: "literal", value: "yesterday"}]
+
+console.log(rtf1.formatToParts(10, 'seconds'));
+// expected output:  [{ type: "literal", value: "in " },
+//                    { type: "integer", value: "10", unit: "second" },
+//                    { type: "literal", value: " seconds" }]
```

Comment:
In both these cases this is not the exact output.

Also though the actual output is very hard to read here:

```
> Array [Object { type: "literal", value: "in " }, Object { type: "integer", value: "10", unit: "second" }, Object { type: "literal", value: " seconds" }]
```

I think it would be more instructive to log the result a piece at a time. This obviously takes up more space, so perhaps just have one example. For example:

```
var rtf1 = new Intl.RelativeTimeFormat('en', { numeric: 'auto' });

var parts = rtf1.formatToParts(10, 'seconds');

console.log(parts[0]);
// expected output: Object { type: "literal", value: "in " }

console.log(parts[1]);
// expected output: Object { type: "integer", value: "10", unit: "second" }

console.log(parts[2]);
// expected output: Object { type: "literal", value: " seconds" }
```

Or maybe even better:

```
var rtf1 = new Intl.RelativeTimeFormat('en', { numeric: 'auto' });

var parts = rtf1.formatToParts(10, 'seconds');

console.log(parts[0].value);
// expected output: "in "

console.log(parts[1].value);
// expected output: "10"

console.log(parts[2].value);
// expected output: " seconds"
```